### PR TITLE
PL-89

### PR DESCRIPTION
**Pull request by AndreyEliVar**
Pl-89 Pull Request

### CODE REVIEW

File: styles/AddHoursEmployee.module.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+.head {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin: 20px 60px;
+    gap: 23px;
+    width: 1149px;
+    height: 50px;
+}
```
Comment (julian-solisaguilar) [inline styles/AddHoursEmployee.module.css:47]: Add EOF

File: pages/api/addHoursEmployee.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        where: {
+            cedulaJuridica
+        },
+        select: {
+            cedulaEmpleador: true
+        }
+    })).cedulaEmpleador;
+
+    
+
```
Comment (julian-solisaguilar) [inline pages/api/addHoursEmployee.js:35]: Fix document formatting, happens in other files as well

Comment (AndreyEliVar) [reply pages/api/addHoursEmployee.js]: fixed up

File: styles/AddHoursEmployee.module.css (hunk 1 above)
Comment (AndreyEliVar) [reply styles/AddHoursEmployee.module.css]: fixed up

**Comment by julian-solisaguilar** [comment]
Denegado 😡

**Review by daar2510 — CHANGES_REQUESTED** [review]
Please make sure you don't have outdated and unused files

**Pull request closed without merging**